x-pack/plugins/monitoring/server/lib/cluster/get_clusters_from_request.ts

```diff
@@ -96,13 +96,14 @@ export async function getClustersFromRequest(
       cluster.ml = { jobs: mlJobs };
     }
 
-    cluster.logs = isInCodePath(codePaths, [CODE_PATH_LOGS])
-      ? await getLogTypes(req, filebeatIndexPattern, {
-          clusterUuid: get(cluster, 'elasticsearch.cluster.id', cluster.cluster_uuid),
-          start,
-          end,
-        })
-      : [];
+    cluster.logs =
+      start && end && isInCodePath(codePaths, [CODE_PATH_LOGS])
+        ? await getLogTypes(req, filebeatIndexPattern, {
+            clusterUuid: get(cluster, 'elasticsearch.cluster.id', cluster.cluster_uuid),
+            start,
+            end,
+          })
+        : [];
```

Comment:
Interesting that `[]` doesn't match the return signature of `getLogTypes` but we can figure that out in future cleanup I'm sure.